AdyenCard/Components/Card/CardViewControllerItemsProvider.swift

```diff
@@ -161,6 +161,41 @@ extension CardViewController {
             return holderNameItem
         }()
 
+        internal lazy var coBadgedCardItem: FormCoBadgedCardItem = {
+            FormCoBadgedCardItem(
+                title: localizedString(
+                    LocalizationKey(key: "Card brand selection (optional)"),
+                    configuration.localizationParameters
+                ),
+                subtitle: localizedString(
+                    LocalizationKey(key: "Select your preferred card brand"),
+                    configuration.localizationParameters
+                ),
+                selectableFormItems: [],
+                style: .init(
+                    title: configuration.style.sectionHeader,
+                    subtitle: configuration.style.footnoteLabel
+                )
+            )
+        }()
+
+        internal func selectableFormItems(from brands: [CardBrand]) -> [SelectableFormItem] {
+            brands.map { brand in
+                let selectableItem = SelectableFormItem(
+                    title: brand.type.name,
+                    imageUrl: numberContainerItem.numberItem.cardTypeLogos.filter { $0.type == brand.type }.first?.url,
+                    isSelected: brand.type.rawValue == brands.first?.type.rawValue ? true : false,
+                    style: .init(title: configuration.style.textField.title),
+                    identifier: brand.type.rawValue
+                )
+                selectableItem.selectionHandler = { [weak self, weak selectableItem] in
+                    guard let self else { return }
+                    handleSelection(identifier: selectableItem?.identifier, brand: brand)
```

Comment:
Does it make sense to call the function without having a selectableItem.identifier?

Reply:
I have removed the identifier from this method.